.github/ISSUE_TEMPLATE/create_issue_euc_update.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Feature Request
```

Comment:
![image](https://user-images.githubusercontent.com/14352021/210888268-f5f3dadb-3442-46b8-8f12-7a7bc5b403fe.png)

Can you please review the way this content gets added to the issue? It's not formatted as expected.